subxt/src/rpc.rs

```diff
@@ -176,16 +176,17 @@ impl From<U256> for NumberOrHex {
 }
 
 /// An error type that signals an out-of-range conversion attempt.
-#[derive(thiserror::Error)]
-pub struct TryFromIntError(pub(crate) ());
+#[derive(Debug, thiserror::Error)]
+#[error("Out-of-range conversion attempt")]
+pub struct TryFromIntError();
```

Comment:
```suggestion
pub struct TryFromIntError;
```
Redundant `()`, would need to fix usages

Reply:
Thx, interesting that `clippy` doesn't flag it.